pallets/randomness/src/benchmarking.rs

```diff
@@ -0,0 +1,32 @@
+#![cfg(feature = "runtime-benchmarks")]
+
+use frame_benchmarking::v2::*;
+use frame_system::RawOrigin;
+
+use super::*;
+#[allow(unused)]
+use crate::Pallet as RandomnessPallet;
+
+#[frame_benchmarking::v2::benchmarks(
+    where T: crate::Config
+)]
+mod benchmarks {
+    use super::*;
+
+    /// The worst case scenario is when the history is full.
+    /// Instead of just adding the randomness value, it requires removing the oldest value too.
+    #[benchmark]
```

Comment:
Could you explain how does this work? I mean, I can't see how we are filling up history here.